DH_Engine/Classes/DHVoiceChatRoom.uc

```diff
@@ -11,7 +11,7 @@ var int SquadIndex;
 // NOTE: overridden to eliminate "has left channel" chat messages
 function RemoveMember(PlayerReplicationInfo PRI)
 {
-    if (PRI != none && PRI.VoiceID < 32 && IsMember(PRI, true))
+    if (PRI != none && PRI.VoiceID < 255 && IsMember(PRI, true))
```

Comment:
I made it < 255 because I think somewhere I saw 255 as some null value being used.